actions/views/create_comment.tsx

```diff
@@ -72,7 +73,7 @@ export function makeOnMoveHistoryIndex(rootId: string, direction: number) {
             dispatch(moveHistoryIndexForward(Posts.MESSAGE_TYPES.COMMENT as 'comment'));
         }
 
-        const nextMessageInHistory = getMessageInHistory(getState());
+        const nextMessageInHistory = getMessageInHistory(state);
```

Comment:
I'm a bit skeptical on this one, are we sure that we don't want to pass in `getMessageInHistory` the updated `state`, do the above actions change what's returned?